phabricatoremails/settings.py

```diff
@@ -19,6 +19,9 @@
 from sqlalchemy import create_engine
 
 
+SETTINGS_FILE_ENVIRONMENT_VARIABLE = "PHABRICATOR_EMAILS_SETTINGS_FILE"
```

Comment:
For consistency, this could be renamed to `SETTINGS_PATH_ENV_KEY` and `PHABRICATOR_EMAILS_SETTINGS_PATH` respectively, or something similar, similar to what you have in `load` (`environment_path`), though I would suggest renaming that to `settings_path` (keyword here being `path` to make it clearer that this is a path and not a file object).